bin/init.sh

```diff
@@ -99,21 +99,21 @@ if [ ! -f vendor/envoy-$PROXYVERSION ] ; then
         exit 1
     fi
 
-    echo "Downloading envoy $PROXY $PROXYVERSION using $DOWNLOAD_COMMAND"
-    ${DOWNLOAD_COMMAND} https://storage.googleapis.com/istio-build/proxy/envoy-$PROXY.tar.gz | tar xz
-    cp usr/local/bin/envoy $ISTIO_GO/vendor/envoy-$PROXYVERSION
+    time ${DOWNLOAD_COMMAND} https://storage.googleapis.com/istio-build/proxy/envoy-$PROXY.tar.gz | tar xz
+    echo "Downloading envoy $PROXY using $DOWNLOAD_COMMAND"
```

Comment:
please put the echo before the command not after so we know what it's doing/how it fails (#3231)

Reply:
Ok.